src/hooks/useClaim.ts

```diff
@@ -14,9 +15,14 @@ export interface TxHistory {
 
 const useClaim = () => {
   const { walletCurrentAddress } = useRainbowContext()
+  const { txType, withDrawStep } = useBridgeStore()
 
   const { comboPageTransactions, pageTransactions, generateTransactions } = useClaimStore()
 
+  const isOnClaimPage = useMemo(() => {
+    return txType === "Withdraw" && withDrawStep === "2"
```

Comment:
I rembemer that when the `mode` was changed from `Transaction` to `History`, we didn't deal with`txType` and `withDrawStep`, so maybe `mode==="Transaction"&& txType === "Withdraw" && withDrawStep === "2"` is more appropriate?